src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1071,6 +1071,11 @@
       "run": "after",
       "snippetPath": "addAuthoritySourceFiles.sql",
       "fromModuleVersion": "25.0.1"
+    },
+    {
+      "run": "after",
+      "snippetPath": "setDefaultMetadataForHrIdHoldings.sql",
```

Comment:
Hello @RomanChernetskyi, propose the script name something like this "setDefaultMetadataForHrIdSettings" because the script updates metadata not only for holdings hrid settings but for other entity types